scripts/dev-setup.sh

```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/env bash
+
+set -e
+set -x
+
+echo "============================================================"
+echo "==      BUILDING DOCKER IMAGES FOR LOCAL DEVELOPMENT      =="
+echo "============================================================"
+
+# Build base image if required
+if ! (docker images | grep dizk-base) ; then
+    docker build -t dizk-base -f Dockerfile-base .
+fi
+
+# Build dizk development image if required
+if ! (docker images | grep dizk-dev) ; then
+    docker build -t dizk-dev -f Dockerfile-dev .
+fi
```

Comment:
Wouldn't it be safer to simply rely on docker's caching here? If an imagine exists, all its layers will be available/cached and the build will be very fast. I get that we may want to avoid this step if the image exists already, but running this script after modifying the content of Dockerfile-base/dev will not build the new image, and may turn out to be misleading.

Reply:
changd.  (I was originally hoping to rely on this, but when I tried it on my machine I saw a full rebuild.  Evidently I did something wrong as it does now indeed seem to work as expected.)

Reply:
Maybe the "full rebuild" was because you deleted your images, or simply changed the Dockerfile? (e.g. the base image in the `FROM`) Not sure.. good if it works now!